lib/consumerGroup.js

```diff
@@ -611,6 +611,48 @@ ConsumerGroup.prototype.sendOffsetCommitRequest = function (commits, cb) {
   }
 };
 
+ConsumerGroup.prototype.addTopics = function (topics, cb) {
+  topics = Array.isArray(topics) ? topics : [topics];
+  let self = this;
+
+  if (!this.client.ready) {
+    this.client.once('ready', this.addTopics(topics, cb));
```

Comment:
should wrap the addTopics call in a fat arrow function.

Reply:
done()